### PR TITLE
feat(imjoy-api): add viewer wrapper stub

### PR DESCRIPTION
**Pull request by PaulHax**
@bnmajor This could be a start.

Related to #9

### CODE REVIEW

File: packages/imjoy-api/src/viewer.cy.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+const serverUrl = 'http://localhost:9000';
+
+describe('imjoy-viewer', () => {
+  it('registers service with Hypha server', async () => {
+    const viewer = createViewer();
+
+    // TODO? beforeEach spin up hypha signaling server
+    const server = await hyphaWebsocketClient.connectToServer({
+      name: 'remote-renderer-client',
```
Comment (thewtex) [inline packages/imjoy-api/src/viewer.cy.ts:13]: test-client

File: packages/imjoy-api/src/viewer.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+};
+
+export async function setup(server: HyphaServer, viewer: Viewer) {
+  const service = await server.registerService({
+    name: "itk_viewer",
+    id: "itk-viewer",
+    description: "Retrieve the viewer API",
+    viewer: () => viewer,
+    addViewport: (viewport: Viewport, name: string) => viewer.send({ type: 'addViewport', viewport: viewport, name: name }),
+    setImage: (image: MultiscaleSpatialImage, name?: string) => viewer.send({ type: 'setImage', image, name: name ?? 'image' }),
```
Comment (PaulHax) [inline packages/imjoy-api/src/viewer.ts:18]: Right now, perhaps all we need as an RPC API for Viewer actor is:
```
create: () => {
  viewerActor = createViewer()
 // returns nothing  (maybe an ID in future to distriquish bettween mulipble client side viewers?)
}
setImage: (zarrStore: ZarrStore, name: string)  => viewer.send({ type: 'setImage', image, name: name ?? 'image' }),
}
```

Comment (bnmajor) [reply packages/imjoy-api/src/viewer.ts:18]: Passing in an existing `Viewer` to setup does work if we want to continue to support that as well? Adding `create` is certainly an option as well though! I think I was just assuming that the `Viewer` may already exist on setup but that may not be the case.

Comment (PaulHax) [reply packages/imjoy-api/src/viewer.ts:18]: Good point.  Could drop `create` and assume each imjoy service has one to one mapping with a Viewer.`

**Pull request closed without merging**